### PR TITLE
[BUG] 245 uncover text

### PR DESCRIPTION
**Pull request by laurenmcp**
## [BUG] 245 uncover text

Issue Number(s): #245 .

What does this PR change and why?

Update cover/uncover alert text to include consequences on volunteer ability to submit foster offer

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria are met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #number_of_pr

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: web/components/PetPostModal/MarkCoveredModal.tsx [hunk 1]
```diff
@@ -43,8 +43,8 @@ function MarkCoveredModal(props: {
           </ModalHeader>
           <ModalBody>
             {isCovered
-              ? "Are you sure you would like to uncover this post? This post would show up on the feed if uncovered."
-              : "Are you sure you would like to mark this post as covered? This post would be hidden on the feed if covered."}
+              ? "Are you sure you would like to uncover this post? This post would show up on the feed if uncovered, and volunteers would be able to resubmit foster offers."
+              : "Are you sure you would like to mark this post as covered? This post would be hidden on the feed if covered, and volunteers would not be able to submit foster offers."}
```
Comment (katsukixyz) [inline web/components/PetPostModal/MarkCoveredModal.tsx]: Nit: grammatical changes

Uncover: "Are you sure you would like to uncover this post? This post will go to the top of the feed and become visible on the feed to volunteers. Volunteers will be able to resubmit offers, even if they have submitted offers for this dog previously."

Cover: "Are you sure you would like to mark this post as covered? This post will become hidden on the feed to volunteers, and no further offers for this dog will be possible until this post is uncovered."

**Review by katsukixyz — CHANGES_REQUESTED** [review]

**Review by katsukixyz — APPROVED** [review]
LGTM!
